plugins/woocommerce-blocks/tests/e2e/tests/price-filter/price-filter.block_theme.side_effects.spec.ts

```diff
@@ -116,7 +142,7 @@ test.describe( `${ blockData.name } Block - with PHP classic template`, () => {
 			postType: 'wp_template',
 		} );
 
-		await editor.canvas.click( 'body' );
+		await editor.canvas.locator( 'body' ).click();
```

Comment:
In other areas, we've changed `editor.canvas.ACTION` to `page.ACTION`. Shouldn't we do the same here and in all other sections where we're currently still using `editor.canvas.ACTION`?

Reply:
Good question. With [this PR](https://github.com/WordPress/gutenberg/pull/54911), the canvas now returns a frameLocator. [According to Playwright documentation](https://playwright.dev/docs/api/class-framelocator): 

> FrameLocator represents a view to the iframe on the page. It captures the logic sufficient to retrieve the iframe and locate elements in that iframe

In some cases, we want to ensure that the actions are executed on the iframe instance and not on the main document. Does it make sense?

Reply:
I had to give it a few reads, but I think I get it now. With `page.ACTION` we can only inspect what's on the actual page, while `editor.canvas.ACTION` allows us to inspect what's inside an iframe. I wasn't aware of that yet.